selfdoc/Plugin/TwigFilterClassParser/TwigFilterClassParserPlugin.php

```diff
@@ -10,9 +10,9 @@
 use BumbleDocGen\Core\Plugin\PluginInterface;
 use BumbleDocGen\Core\Renderer\Context\RendererContext;
 use BumbleDocGen\Core\Renderer\Twig\Filter\CustomFilterInterface;
```

Comment:
Is there any value in moving this to a CorePlugin?

Reply:
These don't make sense, because they only relate to the documentation of the documentation generator itself :)